<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintPointDialog</class>
 <widget class="QDialog" name="PrintPointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>506</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>506</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="about_button">
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>18</y>
     <width>505</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>584</y>
     <width>505</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>180</y>
     <width>497</width>
     <height>407</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Point Information</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>30</y>
       <width>416</width>
       <height>321</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>414</width>
        <height>319</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="prev_button">
     <property name="geometry">
      <rect>
       <x>323</x>
       <y>354</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fit_button">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>132</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources_pasture.qrc">
       <normaloff>:/plugins/lm2_pasture/zoom_fit.png</normaloff>:/plugins/lm2_pasture/zoom_fit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>44</y>
       <width>381</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Viewer</string>
     </property>
    </widget>
    <widget class="QLabel" name="count_label">
     <property name="geometry">
      <rect>
       <x>273</x>
       <y>357</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cover_rbutton">
     <property name="geometry">
      <rect>
       <x>224</x>
       <y>6</y>
       <width>75</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomout_button">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>81</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources_pasture.qrc">
       <normaloff>:/plugins/lm2_pasture/zoom_out.png</normaloff>:/plugins/lm2_pasture/zoom_out.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="load_image_button">
     <property name="geometry">
      <rect>
       <x>404</x>
       <y>4</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="around_rbutton">
     <property name="geometry">
      <rect>
       <x>314</x>
       <y>6</y>
       <width>75</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Around</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomin_button">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources_pasture.qrc">
       <normaloff>:/plugins/lm2_pasture/zoom_in.png</normaloff>:/plugins/lm2_pasture/zoom_in.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="next_button">
     <property name="geometry">
      <rect>
       <x>403</x>
       <y>354</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="print_button">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>183</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources_pasture.qrc">
       <normaloff>:/plugins/lm2_pasture/printer.png</normaloff>:/plugins/lm2_pasture/printer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Pasture Information</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>481</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Calculate Values</string>
     </property>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Боломжит даац</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>21</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1 га-д бэлчиж болох хонин толгой</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>81</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D-2:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="calc_d1_100ga_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="calc_unelgee_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>139</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_47">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>51</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>100 га-д бэлчиж болох хонин толгой</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="calc_d1_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>111</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D-3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Unelgee:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="calc_d2_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_51">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Боломжит ХТ- Одоо байгаа ХТ = Үнэлгээ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D-1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_48">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1 хонин толгойд шаардлагатай талбай</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>51</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>D-1-100ga:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="calc_d3_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="calc_duration_sbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>1000000000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>172</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Duration:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>172</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Мал бэлчээрлэх хугацаа /хоногоор/</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>220</y>
       <width>481</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Print Pasture</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Print</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>148</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>БАХ-н бэлчээр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>Улирлын бэлчээр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Сэргэх чадавхийн зураг</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="point_address_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>28</y>
     <width>496</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="land_form_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>496</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Land Form</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>7</y>
     <width>330</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="year_sbox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>93</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1900</number>
   </property>
   <property name="maximum">
    <number>2100</number>
   </property>
   <property name="value">
    <number>2017</number>
   </property>
  </widget>
  <widget class="QPushButton" name="load_data_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>92</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Recovery Class</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bio Mass</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Cover</string>
   </property>
  </widget>
  <widget class="QLabel" name="biomass_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Bio Mass</string>
   </property>
  </widget>
  <widget class="QLabel" name="rc_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Recovery Class</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="cover_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>Total Cover</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources_pasture.qrc"/>
 </resources>
 <connections/>
</ui>
